website/docs/patterns/configuring_plugins.md

```diff
@@ -0,0 +1,127 @@
+---
+id: configuring_plugins
+title: Configuring Plugins
+---
+
+import {ExampleGithubLink} from "@site/src/components/GithubLink"
+
+
+All Hydra plugins come with sensible defaults which work out of the box with the example applications.
+You can override/extend the plugins defaults by: 1) overriding in primary config or 2) extending 
+the default plugin config.
+
+:::info
+This page assumes that you are familiar with the contents of [Common Patterns/Extending Configs](patterns/extending_configs.md).
+:::
+
+The following methods apply to all Hydra plugins. We use `BasicSweeper` from `hydra-core` here as an example,
+in particular we try to override  `hydra.sweeper.max_batch_size` config, which controls how many jobs the sweeper
+launches in each batch.
+
+Here we run `my_app.py` before any overrides, notice that the Sweeper launches all jobs in one batch.
+
+<div className="row">
+<div className="col col--2">
+
+```yaml title="config.yaml" 
+a: 1
+
+
+
+
+
+
+```
+</div>
+<div className="col  col--5">
+
+```python title="my_app.py" 
+@hydra.main(config_name="config")
+def my_app(_cfg: DictConfig) -> None:
+    pass
+
+
+if __name__ == "__main__":
+    my_app()
+```
+</div>
+<div className="col  col--5">
+
+```commandline title="python my_app.py -m a=1,2,3" 
+[HYDRA] Launching 3 jobs locally
+[HYDRA]        #0 : a=1
+[HYDRA]        #1 : a=2
+[HYDRA]        #2 : a=3
+
+
+
+```
+</div>
+</div>
+
+
+
+### Overriding in primary config
+<ExampleGithubLink text="Example application" to="examples/patterns/configuring_plugins_primary_config"/>
+
+In the primary config, we override `hydra.sweeper.max_batch_size`.
+Running the application, we can see the the jobs are launcher in batch of one.
+
+<div className="row">
+<div className="col col--6">
+
+```yaml title="config.yaml" 
+a: 1
+
+hydra:
+  sweeper:
+    max_batch_size: 1
+
+
+```
+</div>
+
+<div className="col  col--6">
+
+```commandline title="python my_app.py -m a=1,2,3" 
+[HYDRA] Launching 1 jobs locally
+[HYDRA]        #0 : a=1
+[HYDRA] Launching 1 jobs locally
+[HYDRA]        #1 : a=2
+[HYDRA] Launching 1 jobs locally
+[HYDRA]        #2 : a=3
+
+```
+</div>
+</div>
+
+
+
+### Extending plugin default config
+
+<ExampleGithubLink text="Example application" to="examples/patterns/configuring_plugins_extend_defaults"/>
+
+To achieve the same configuration, we can also create a separate config for the sweeper which helps keeping the primary config clean.
+
+<div className="row">
+<div className="col col--6">
+
+```yaml title="hydra/sweeper/custom_sweeper.yaml" 
+defaults:
+  - basic
+
+max_batch_size: 1
```

Comment:
batch of 2 may make the difference more obvious. took me a moment to see the difference in output.

Reply:
True, I've updated the example.